peak_finder/application.py

```diff
@@ -2001,6 +1998,21 @@ def trigger_click(  # pylint: disable=too-many-arguments, too-many-locals
 
         return ["Saved to " + str(workspace.h5file)]
 
+    @property
+    def params(self) -> PeakFinderParams:
+        """
+        Application parameters
+        """
+        return self._params
+
+    @params.setter
+    def params(self, params: PeakFinderParams):
+        assert isinstance(
```

Comment:
raise TypeError instead of assert

Reply:
Nice catch. Copied this over without really looking.